tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -199,8 +200,9 @@
     },
     _maybeUpdateTool: function(datasets, selectedDataset, currentTool) {
       if (currentTool == "trace_viewer") {
-        var trace_data_url = (getRouter().pluginRunTagRoute('profile', '/data')(
-            'trace_viewer', datasets[selectedDataset].name));
+        var trace_data_url = addParams(
+            getRouter().pluginRoute('profile', '/data'),
+            {tag: 'trace_viewer', run: datasets[selectedDataset].name});
```

Comment:
Side commentary: It's interesting to see how different plugins take advantage of tags. Some use them for organization. Others apparently place magic strings in them. The debugger stores op names within tags ... We'll see where this goes.